### ISSUE TITLE
Unable to reconnect powerline

### ISSUE THREAD

**Issue by sunghoonhong** [rte-france/Grid2Op#54]
Thanks for releasing new version 0.6.0.

By the way, it seems that reconnection of powerline doesn't work now.
Here is a reproducing code as following.
```
env = grid2op.make('case5_example')
obs = env.reset()
obs, _, done, info = env.step(env.action_space.disconnect_powerline(2))
print(info)
print(obs.line_status)
```
It gives the following output as normal.
>{'disc_lines': array([False, False, False, False, False, False, False, False]), 'is_illegal': False, 'is_ambiguous': False, 'is_dispatching_illegal': False, 'is_illegal_reco': False, 'exception': [], 'rewards': {}}
>[ True  True False  True  True  True  True  True]

Then,
```
obs, _, done, info = env.step(env.action_space.reconnect_powerline(2, 1, 1))
print(info)
print(obs.line_status)
```
It gives the illegal action sign as the following.
>{'disc_lines': array([False, False, False, False, False, False, False, False]), 'is_illegal': True, 'is_ambiguous': False, 'is_dispatching_illegal': False, 'is_illegal_reco': False, 'exception': [Grid2OpException IllegalAction IllegalAction('BaseAction illegal')], 'rewards': {}}
>[ True  True False  True  True  True  True  True]

It seems that it is legal action above situation, but the output tells it is illegal action.
Is it intended outcome?

Best
Sunghoon

**Comment by Tezirg** [comment]
Hello Sunghoon, thank you for reporting this issue.
We already noticed the problem and are working towards a resolution. 

This is expected behavior for connected lines as shown in [this unit test](https://github.com/rte-france/Grid2Op/blob/master/grid2op/tests/test_Rules.py#L444), but grid2op **should not** count a line set bus operation as a substation operation if the line is disconnected. 

One workaround is to allow two substation changes per timestep:
```
from grid2op.Parameters import Parameters
custom_params = Parameters()
custom_params.MAX_SUB_CHANGED = 2
env = grid2op.make('case5_example', param=custom_params)
```
An other workaround is to use the `AlwaysLegal` gamesrules:
```
from grid2op.Rules import AlwaysLegal
env = grid2op.make('case5_example', gamerules_class=AlwaysLegal)
```

**Comment by Tezirg** [comment]
This should be fixed in `0.6.1`.

Feel free to reopen the issue if the problem persists
